coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -1,10 +1,11 @@
 # Legislation evolutions
 
-Openfisca handles the fact that the legislation change over time.
+Openfisca handles the fact that the legislation changes over time.
 
 ## Parameter evolution
 
-Many legislation parameters are regularly re-evaluated. In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.
+Many legislation parameters are regularly re-evaluated. 
+In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.
```

Comment:
> can we simplify this part ? 
""" 
Many legislation parameters are regularly re-evaluated. This is why there is a start date attached to each parameters definition.
In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.

>Example: a user can reflect a change in the `taxes` parameter without altering the code of the `flat_tax_on_salary` formula that uses that parameter.
"""
**To add/change a parameter :** 
1. Open the file where the parameter is described

```xml
<NODE code="taxes">
    <NODE code='salary'>
      <CODE code="rate" description="Rate for the flat tax on salaries">
        <VALUE deb="2016-01-01" valeur="0.25" />
        <VALUE deb="2015-01-01" valeur="0.20" />
      </CODE>
    </NODE>
</NODE>
```
2. Change the parameter with its new start date
```xml
<NODE code="taxes">
    <NODE code='salary'>
      <CODE code="rate" description="Rate for the flat tax on salaries">
        <VALUE deb="2017-01-01" valeur="0.30" />
        <VALUE deb="2016-01-01" valeur="0.25" />
        <VALUE deb="2015-01-01" valeur="0.20" />
      </CODE>
    </NODE>
</NODE>
```
3. Conclusion

After this change : 
`legislation(period).path.to.taxes_parameter` will return the corresponding number
`legislation(2015-01).path.to.taxes_parameter` will return `0.2`
`legislation(2017-01).path.to.taxes_parameter` will return `0.3`